### ISSUE TITLE
Change pixel format with using emulator 

### ISSUE THREAD

**Issue by oak-tree** [basler/pypylon#108]
Hey, 
I'm trying to make my images on emulator colorful. As suggested in Basler docs, I changed the `camera.PixelFormat`.

## What I did:
In order to do so first, I fetched, the available format from emulator using: 
`camera.PixelFormat.Symbolics` and got 
`('Mono8', 'Mono16', 'BGRA8Packed', 'BGR8Packed', 'RGB8Packed', 'RGB16Packed')`
`camera.PixelFormat.SetValue(<the value>)

## The problem
Then, except the `Mono8` value, When I try to use the `colorful` or `Mono16` formats I get 
`Process finished with exit code -1073740791 (0xC0000409)`  as soon as I run the command         `camera.StartGrabbing(pylon.GrabStrategy_LatestImageOnly)`

It does work the `Mono8` values. 
Any ideas about how to overcome and use colorful images within the emulator?

Thanks,

## Extra code
  ```
camera.ImageFilename = img_dir
  camera.ImageFileMode = "On"

  # disable testpattern [ image file is "real-image"]
  camera.TestImageSelector = "Off"
  camera.Height.SetValue(416);
  camera.Width.SetValue(1248);

  
  camera.PixelFormat.SetValue("BGRA8Packed")
 camera.StartGrabbing(pylon.GrabStrategy_LatestImageOnly
```

## Edit
I tried to do that from `Pylon Viewer`. Changing the `Pixel format` does work from `viewer` and I can see the colorful images. But it seems that when changing the `Pixel format` at the viewer, it is automatedly changing the `PixelSize` as well. 

But on python when I do:
`camera.PixelFormat.SetValue("BGRA8Packed")` and `camera.PixelSize.GetValue()` stays the same. Note that it is not possible to do `camera.PixelSize.SetValue(<X>)`

**Comment by thiesmoeller** [comment]
The payloadsize is not updated correctly. Only when Pixelformat is read once ....
Fixed in pylon >= 5.2

But we have a workaround for you:

```
camera.PixelFormat.SetValue("BGRA8Packed")
camera.PixelFormat.GetValue()
camera.StartGrabbing(pylon.GrabStrategy_LatestImageOnly) 
...

```

**Comment by oak-tree** [comment]
@thiesmoeller thanks! the reading the `PixelFormat` once does the trick
